src/MacroTools/Powers/Rematerialization.cs

```diff
@@ -50,10 +50,12 @@ private void OnUnitDeath()
       if (GetRandomReal(0, 1) > _chance 
           || !EligibilityCondition(dyingUnit) 
           || _noReturnRect.Contains(dyingUnit.GetPosition())
+          || dyingUnit.IsType(UNIT_TYPE_RESISTANT)
```

Comment:
Let's update the Description accordingly, so like "Your non-Resistant units have a {chance*100}% chance to rematerialize at {returnPointName} upon death.";